app/assets/src/styles/samples.scss

```diff
@@ -43,15 +45,10 @@ $break-large: 1070px;
 .tab-screen-content {
   .content-title{
     color: #313E49;
-    font-size: 16px;
+    font-size: $font-size-h5;
     font-weight: 600;
```

Comment:
Line 49 in Samples.scss

Should then be:
`font-weight: $headings-font-weight;`